boilerplate/_lib/ensure.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo pipefail
+
+GOLANGCI_LINT_VERSION="1.30.0"
+DEPENDENCY=${1:-}
+
+case "${DEPENDENCY}" in
+    golangci-lint)
```

Comment:
wouldn't hurt to have a default case that blows up if you pass a dep we don't understand (or misspell one we do).